CMSgov/onemac
# Oy2 20446

Story: https://qmacbis.atlassian.net/browse/OY2-20446
Endpoint: See github-actions bot comment

### Details

As a user I want to be clearly notified that I only need to create one submission for multiple underlying numbers.

### Changes

- For Appk forms separate and bold this part of the header text: "If your Appendix K submission is for more than one waiver number, please enter one of the applicable waiver numbers. You do not need to create multiple submissions.”

### Implementation Notes

- Fixed in both Submission and Package view

### Test Plan

1. Login
2. Navigate to "Create a 1915(c) Appendix K Amendment" in package view or "Submit 1915(c) Appendix K Amendment" in submission view
3. Confirm text is formatted as it should be
